e2e/cypress/integration/channel/channel_settings_spec.js

```diff
@@ -52,4 +54,32 @@ describe('Channel Settings', () => {
             cy.get('@channel').find('span.btn-close').should('exist');
         });
     });
+
+    it('MM-T882 Channel settings / Channel URL validation works properly', () => {
+        // # Login as admin and create new channel
+        cy.apiAdminLogin();
```

Comment:
Note: It's fine not to login as admin, and remain with the regular user in session.

Reply:
can't change the team name without the admin perhaps I am missing a setting?

Reply:
I see but as much as possible, please don't test using sysadmin user. With that, what I meant was you may use the channel created by `cy.apiInitSetup`,
```javascript
let testChannel;

before(() => {
    cy.apiInitSetup().then(({team, user, channel}) => {
        testTeam = team;
        testChannel = channel;
        ...
    });
}); 
```
Then use that channel instead of creating another one. But I'm good as is, only something to look forward.

Reply:
Oh yeah I used off-topic and it worked!